### PR TITLE
Increase wait time between builds for embedded runner

### PR DESCRIPTION
**Pull request by wolfs**
So the file timestamps can be used to distinguish between the output
files since the optimized incremental Java compilation now made
builds too fast so this is a problem now.

Fixes gradle/gradle-private#3364

### CODE REVIEW

**Comment by wolfs** [comment]
@bot-gradle test QF

File: subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/FileSystemWatchingHelper.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 public class FileSystemWatchingHelper {
 
-    private static final int WAIT_FOR_CHANGES_PICKED_UP_MILLIS = 120;
+    private static final int WAIT_FOR_CHANGES_PICKED_UP_MILLIS = 500;
```
Comment (melix) [inline subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/FileSystemWatchingHelper.java:25]: Maybe this should be dependent on the JDK? Like 500ms for Java 8 given what you've found, but keep 120 for 11+?

**Review by melix — APPROVED** [review]

File: subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/FileSystemWatchingHelper.java (hunk 1 above)
Comment (wolfs) [reply subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/FileSystemWatchingHelper.java]: Good point, I did that!

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**